### PR TITLE
Add gql response endpoint

### PR DESCRIPTION
**Pull request by xjules**
Resolves https://github.com/equinor/ert-storage/issues/107

### CODE REVIEW

File: ert_storage/graphql/ensembles.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
-from typing import Iterable, List, Optional, TYPE_CHECKING
+from ert_storage.database_schema import record
+from ert_storage.database_schema.record import RecordClass
+from typing import Iterable, List, Mapping, Optional, TYPE_CHECKING
```
Comment (oysteoh) [inline ert_storage/graphql/ensembles.py:3]: Mapping not used

File: ert_storage/graphql/ensembles.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-from typing import Iterable, List, Optional, TYPE_CHECKING
+from ert_storage.database_schema import record
+from ert_storage.database_schema.record import RecordClass
```
Comment (oysteoh) [inline ert_storage/graphql/ensembles.py:2]: Both line 1 and 2 are not needed

File: ert_storage/graphql/ensembles.py [hunk 3]
```diff
@@ -33,6 +40,31 @@ def resolve_parent_ensemble(
             return update.ensemble_reference
         return None
 
+    def resolve_unique_responses(
+        root: ds.Ensemble, info: "ResolveInfo"
+    ) -> Iterable[ds.Record]:
+        session = info.context["session"]  # type: ignore
```
Comment (oysteoh) [inline ert_storage/graphql/ensembles.py:46]: what do you mean with the comment?

File: tests/integration/gql/test_responses.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
+import random
+import uuid
+import io
+from fastapi import params
+import numpy as np
+from numpy.testing import assert_equal
+import pandas as pd
```
Comment (oysteoh) [inline tests/integration/gql/test_responses.py:7]: only numpy and pandas in use

File: ert_storage/graphql/ensembles.py (hunk 3 above)
Comment (xjules) [reply ert_storage/graphql/ensembles.py:46]: That's for the `mypy`

**Comment by oysteoh** [comment]
Would prefer one commit introducing Responses in the schema. Don't think it is necessary having a separate commit for the tests. Other than that this looks very good from my point of view! 👍

**Comment by xjules** [comment]
> Would prefer one commit introducing Responses in the schema. Don't think it is necessary having a separate commit for the tests. Other than that this looks very good from my point of view!

Good that you mention the Responses in the schema as it's not required so removing it! Still, I can merge test and gql code.

File: ert_storage/graphql/ensembles.py (hunk 2 above)
Comment (oysteoh) [reply ert_storage/graphql/ensembles.py:2]: These still exists but are not used as i can see?

**Review by oysteoh — APPROVED** [review]

**Pull request merged**